tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -250,6 +250,23 @@ error: Unknown symbol \\"this\\"
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`ambiguous_api_paths.test.w 1`] = `
+"ERROR: Endpoint for path '/test/{variable}' and method 'GET' is ambiguous - it conflicts with existing endpoint for path '/test/path'
+
+../../../examples/tests/invalid/target/test/ambiguous_api_paths.test.wsim.157309.tmp/.wing/preflight.js:38
```

Comment:
We probably need to sanitize the number here: `ambiguous_api_paths.test.wsim.157309.tmp`
(next to this code, I think- https://github.com/winglang/wing/blob/8eab41aec4294c4299124c186f81868f587df591/tools/hangar/src/utils.ts#L53-L55)

since we generate a new temp directory for each build, and we are re-generating the snapshot it doesn't match, and it never matches because of the that number, it keeps re-building over and over (that's the monabot commits you see)